libtsuba/src/RDGManifest.h

```diff
@@ -0,0 +1,151 @@
+#ifndef KATANA_LIBTSUBA_RDGMANIFEST_H_
+#define KATANA_LIBTSUBA_RDGMANIFEST_H_
+
+#include <cstdint>
+#include <regex>
+#include <set>
+
+#include "katana/JSON.h"
+#include "katana/Logging.h"
+#include "katana/Uri.h"
+#include "katana/config.h"
+#include "tsuba/RDGLineage.h"
+#include "tsuba/tsuba.h"
+
+namespace tsuba {
+
+static const char* gDefaultRDGViewType = "rdg";
```

Comment:
Should be `kDefaultRDGViewType`

https://google.github.io/styleguide/cppguide.html#Constant_Names